### ISSUE TITLE
Auto-remove entities removed from Yaml config

### ISSUE THREAD

**Issue by pickonedev** [Wibias/hass-variables#92]
Hi!

I only use YAML configuration with this addon and everytime I delete/remove a variable from the config, it won't dissapear like in the past, I always need to go into the GUI config and delete it from there as well...
There is something which I don't know to do or the new updated addon does not know to delete completely the entity after it was removed from YAML?

It would really help this, because if I delete more variables and I want to create others with similar (but not the same) name, I won't know which is which...

Thank you!

**Comment by Snuffy2** [comment]
Yeah, I had been trying to come up with a way to do this and I think I finally have a working solution. PR coming soon
